test/local/account_routes.js

```diff
@@ -765,61 +765,122 @@ test('/account/login', function (t) {
   var route = getRoute(accountRoutes, '/account/login')
 
   t.test('sign-in confirmation disabled', function (t) {
-    return runTest(route, mockRequest, function (response) {
-      t.equal(mockDB.emailRecord.callCount, 1, 'db.emailRecord was called')
-      t.equal(mockDB.createSessionToken.callCount, 1, 'db.createSessionToken was called')
-      var tokenData = mockDB.createSessionToken.getCall(0).args[0]
-      t.notOk(tokenData.mustVerify, 'sessionToken was created verified')
-      t.notOk(tokenData.tokenVerificationId, 'sessionToken was created verified')
-      t.equal(mockDB.sessions.callCount, 1, 'db.sessions was called')
-
-      t.equal(mockLog.stdout.write.callCount, 1, 'an sqs event was logged')
-      var eventData = JSON.parse(mockLog.stdout.write.getCall(0).args[0])
-      t.equal(eventData.event, 'login', 'it was a login event')
-      t.equal(eventData.data.service, 'sync', 'it was for sync')
-      t.equal(eventData.data.email, TEST_EMAIL, 'it was for the correct email')
-      t.deepEqual(eventData.data.metricsContext, mockRequest.payload.metricsContext, 'it contained the metrics context')
+    t.plan(2)
+    t.test('sign-in does not require verification', function (t) {
```

Comment:
Is it possible to test which emails have been sent to ensure only the expected ones are sent?